### ISSUE TITLE
Planar conductor-loss geometry weight: chain for #120, #101, #121, #103, #123, #124

### ISSUE THREAD

**Issue by gvcallen** [paramrf/paramrf#122]
Tracking issue. Hand this to an agent and it can work the four tickets below end to end without re-deriving the shared context.

## The chain

Work in this order. Each step is independently landable and verifiable; the order is chosen to avoid rework, not because of hard dependencies.

- [x] **1. #120** — `IncrementalInductanceCurrentDistribution`: evaluate Wheeler's recession derivative by `jax.grad` instead of his 1942 closed-form fit. The largest and highest-evidence change; do it first because #121 and #103 both describe what it produces.
- [x] **2. #101** — trace/ground current split as a second peer strategy. Independent of #120, but sequenced after it so that #121 can cover every distribution in one pass and #103 can describe what actually shipped.
- [x] **3. #121** — docstring note on the slab surface-impedance default vs. half-space external solvers. Trivial, but deliberately last of the code tickets: it applies to *every* distribution carrying a `slab_impedance` field, so running it after 1 and 2 covers them all at once instead of three times.
- [x] **4. #103** — the reference page of conductor-loss alternatives not implemented, including the standing record of the sign contradiction below. Last, because it documents the outcome of 1 and 2.
- [ ] **5. #123** — stripline finite-thickness conductor loss and dc resistance floor. Folds the two closed stripline tickets (#91, #102) into one. **Unblocked** — it needs only #100, which landed, so it can be picked up at any point; it sits here because it is the microstrip work's natural follow-on, not because anything gates it.
- [ ] **6. #124** — decide whether stripline charges ground-plane loss separately. Genuinely blocked, by #123 and #101: it is the one piece of the stripline work that needs the microstrip trace/ground split to exist before its reasoning can be written.

Only #124 carries hard blocking edges (#123 and #101, both recorded as native GitHub dependencies). Everything else is startable now, so the order above is about avoiding rework rather than obeying constraints. To parallelise: #120, #101 and #123 can all run at the same time; #121 and #103 should wait for whichever of #120/#101 you intend to land, and #124 waits on #123 and #101.

## Shared context — read once, applies to all four

**Why any of this matters: `k_c` and conductivity are exactly degenerate.** The conductor channel enters `Z` only through the product `Z_s * k_c`, and `Z_s ~ sigma^(-1/2)`. A fit with free `sigma` therefore absorbs *any* `k_c` error silently — a 22% error in `k_c` surfaces as `1.22^2 = 1.49x` copper and nothing else. This is why a wrong geometry weight has survived: it never looks like a bad `k_c`, only like an implausible conductivity. Every ticket here exists because of that.

**The default does not change.** `WheelerCurrentDistribution` stays the microstrip default through all four tickets. #120 and #101 both ship as *selectable peers*. Do not change the default in either, and do not treat "my new one is more accurate" as licence to. The default question is deliberately deferred until the sign question below is settled.

**#120 and #101 disagree on the sign of Wheeler's error.**

| source | Wheeler's error vs. the better answer |
|---|---|
| #120, incremental-inductance derivative | Wheeler is **22% high** at `w/h = 2.5`, corroborated by four independent routes landing in 306-320 against Wheeler's 385.7 |
| #101, trace/ground split | Wheeler is **12-18% low** at strong skin, consistent with Holloway and Kuester's reported 12-30% underprediction |

Both cite Holloway and Kuester; they cannot both be right. #120's evidence is stronger (independent 2D field solve, power-loss surface-impedance integral, and a volumetric PEEC solve). **You are not required to resolve this.** You *are* required not to paper over it: state departures with their sign in docstrings rather than describing anything as simply "more accurate", and let #103 carry the record.

**Provenance of the numbers.** Every ParamRF-internal number across these tickets was re-verified against `0.34.3` at `c1386448` and reproduces exactly. Numbers attributed to external field solvers and commercial 3D tools are labelled as such and cannot be reproduced in-repo — do not turn them into tests.

## House rules that bite on this work

From `CLAUDE.md` and `docs/agents/`:

- **Breaking changes are fine.** No compatibility shims, no deprecation aliases. Prefer the clean design.
- **Do not add yourself as commit author.** No `Co-Authored-By`, no session link, no tool attribution.
- **scikit-rf is guidance, not ground truth.** On a disagreement, find out which side is wrong before touching a tolerance. Never widen a tolerance until the number is explained.
- **Record tolerances per case.** Do not loosen a global tolerance to make one case pass.
- **Physics docstrings** carry a `**Mathematical Formulation**` section and a `References` section citing the source paper. Fields are documented with `#:` comments above them.
- **`build/` and `dist/` are stale copies.** Searches hit them; never read or edit them as source.

Verify with:

```bash
.venv/bin/python -m pytest
.venv/bin/python -c "import pmrf"
```

## Folded in, not lost

**#91 and #102** were stripline tickets closed in the same 2026-09-01 bulk sweep as #101 and #103, and equally unlanded. They overlapped heavily — both concerned `CohnCurrentDistribution`, which still emits a half-space surface impedance in every case and a zero geometry weight whenever thickness is unspecified, so stripline resistance falls to zero at dc even when the thickness is known.

They are superseded by **#123** (the merged physics and dc floor) and **#124** (the ground-plane charging decision, which was the only part needing #101). #91 and #102 stay closed; the split is by blocking edge, so the bulk of the stripline work is startable now instead of waiting behind the microstrip chain.
